test/delete.test.js

```diff
@@ -151,60 +154,63 @@ test('body - delete', t => {
     fastify.delete('/body', bodySchema, function (req, reply) {
       reply.send(req.body)
     })
-    t.pass()
+    t.assert.ok(true)
   } catch (e) {
-    t.fail()
+    t.assert.fail()
   }
 })
 
 fastify.listen({ port: 0 }, err => {
-  t.error(err)
-  t.teardown(() => { fastify.close() })
+  ifError(err)
```

Comment:
`ifError` doesn't exist in `t`?

```suggestion
  t.assert.ifError(err)
```

Reply:
@gurgunday `t` provides the `t.assert.ifError(err)` utility, scoped only to individual test blocks. For global use, such a utility does not exist in `t`. Therefore, it needs to be imported from the `assert`.

Reply:
wondering if we need this we could also do something like this. @gurgunday wdyt?
```js
fastify.listen({ port: 0 }, err => {
  if (err) throw new Error('Fastify failed to start')
  .....
})
```

Reply:
@dancastillo @jsumners any suggestions for me?

Reply:
```js
const assert = require('node:assert')

fastify.listen(0, (error) => {
	assert.ifError(error)
})
```

Reply:
@jsumners done